src/multicall.ts

```diff
@@ -429,18 +421,18 @@ export class Multicall {
         MulticallOptionsCustomJsonRpcProvider
       >();
       if (customProvider.nodeUrl) {
-        ethersProvider = new ethers.providers.JsonRpcProvider(
+        ethersProvider = new ethers.JsonRpcProvider(
           customProvider.nodeUrl
         );
       } else {
-        ethersProvider = ethers.getDefaultProvider();
+        ethersProvider = ethers.getDefaultProvider(1);
```

Comment:
Previously ethers would default to ethereum mainnet, so just maintaining parity here